src/app/page.tsx

```diff
@@ -397,6 +446,27 @@ const Home = () => {
             </div>
           </div>
           <div className="flex items-center justify-center gap-20">
+            <div
+              onClick={() => {
+                if (!deadline) {
+                  setDeadline(!deadline);
+                  handleSteadyDeadline(page.toString());
+                } else {
+                  setDeadline(!deadline);
+                  handleGetSteadies(false, recruit, type, page.toString());
```

Comment:
```suggestion
                setDeadline(!deadline);
                if (!deadline) {
                  handleSteadyDeadline(page.toString());
                } else {
                  handleGetSteadies(false, recruit, type, page.toString());
```
이렇게 바꿔도 잘 동작하지 않을까요?